translations/Marathi/general-rules.md

```diff
@@ -0,0 +1,40 @@
+# General Rules for Translation in Marathi
+
+## Overview
+
+* Marathi is a very versatile & easy-to-use language like English, so translations
+  are also easy.
+
+* Translations can be done in either way, formal or informal, but formal way is preferred
+  for corporate work.
+
+* The translations can be done in any person( first, second or third) according to the 
```

Comment:
As asked earlier, what is the person that YOU would recommend a web app communicating with a Marathi speaking person?